### PR TITLE
Recovery Plan spec changes for ASR service

### PR DESCRIPTION
**Pull request by amitmall987**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@amitmall987__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (ammall). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

**Comment by hovsepm** [comment]
@azuresdkci add to whitelist

**Comment by markcowl** [comment]
@amitmall987 Test failures, please take a look

File: src/ResourceManagement/RecoveryServices/RecoveryServicesManagement/Microsoft.Azure.Management.RecoveryServices.nuget.proj [hunk 1]
```diff
@@ -5,7 +5,7 @@
     Microsoft.Azure.Management.RecoveryServices
     -->
     <SdkNuGetPackage Include="Microsoft.Azure.Management.RecoveryServices">
-      <PackageVersion>1.0.2-preview</PackageVersion>
+      <PackageVersion>1.0.3-preview</PackageVersion>
```
Comment (hovsepm) [inline src/ResourceManagement/RecoveryServices/RecoveryServicesManagement/Microsoft.Azure.Management.RecoveryServices.nuget.proj]: @amitmall987 You have changed client names in hydra specs. this is a breaking change. Please bump up the major version.

**Pull request merged**